dom.bs

```diff
@@ -10070,6 +10112,7 @@ Tab Atkins,
 Takashi Sakamoto,
 Takayoshi Kochi,
 Theresa O'Connor,
+Theodore Dubois,
```

Comment:
Is this related to this pr?

Reply:
Yes: https://github.com/whatwg/dom/issues/392.